node/test/go/dev-mode/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
If I remove this file I won't have a nice `npm run test:integration` command to run.

Given the tests are written in node it might make sense to leave this file. I could always reference the path to mocha in a script or something but it would be a bit inconvenient 🤷

Reply:
There's `package.json` at `node` folder, so if you want to have shortcut for running go integration tests I'll configure script there.

Here we neither introduce new dependencies scope nor new package, so having `package.json` is confusing.

Reply:
IMO it feels weird to have a dev-mode specific testing command in the root node `package.json` file

Reply:
I think it's ok, as we do not introduce any dev-mode Node.js package (as it's actually a Go package), we configure this script at global `package.json`